### PR TITLE
feature/TR-918/support-rtl-inline-interaction

### PR DESCRIPTION
**Pull request by marpesia**
**Related to:** [TR-918](https://oat-sa.atlassian.net/browse/TR-918)

**Description:**

Adapt Inline text interaction to the item’s writing direction.

**Changes:**

- Force drop down to RTL mode on RTL user language by CSS
- Update `package.json` and `package-lock.json` to be compatible with NPM 6

### CODE REVIEW

File: scss/inc/_select2.scss [hunk 1]
```diff
@@ -147,6 +160,19 @@
...
+    @include ltr {
+        direction: ltr;
+    }
+
+    &.rtl {
+        direction: rtl;
+    }
+    &.ltr {
+        direction: ltr;
+    }
```
Comment (meefox) [inline scss/inc/_select2.scss:175]: Do we really need it?
Seems to be you did a good job here: https://github.com/oat-sa/extension-tao-itemqti/pull/1958/files

Comment (marpesia) [reply scss/inc/_select2.scss:175]: Yes, this is for the dropbox at the bottom of the document. It is outside the component, and the other CSS can't make any difference here.
image

**Review by meefox — APPROVED** [review]
- [x] New code is covered by tests
- [x] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by krampstudio — APPROVED** [review]
Tested in TAE. LGTM. :+1: 

- [ ] New code is covered by tests (style only)
- [ ] Tests are running successfully (old and new ones) on my local machine (2 tests that were previously failing are still failing ;/ )
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] <s>New code is not subject to concurrency issues (if applicable)</s>
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**